module/os/linux/zfs/arc_os.c

```diff
@@ -375,6 +382,39 @@ param_set_arc_int(const char *buf, zfs_kernel_param_t *kp)
 
 	return (0);
 }
+
+/* ARGSUSED */
+static int
+arc_hotplug_callback(struct notifier_block *self, unsigned long action,
+    void *arg)
```

Comment:
Perhaps a comment mentioning these values really should remain in sync with `arc_init()`.